src/content/zh-CN/2019/accessibility.md

```diff
@@ -0,0 +1,309 @@
+---
+part\_number: II
+chapter\_number: 9
+title: 无障碍可访问性
+description: 2019 Web Almanac 网络年鉴的无障碍易访问性章节，涵盖易阅读、媒体、易于导航和与辅助技术的兼容性。
```

Comment:
```suggestion
description: 2019 Web Almanac 网络年鉴的无障碍易访问性章节、涵盖易阅读、媒体、易于导航和与辅助技术的兼容性。
```

Reply:
It should be a comma or a full stop.

Reply:
But which comma? Which one does Chinese use?

Reply:
different meaning in Chinese :  https://www.thoughtco.com/chinese-punctuation-marks-2279717         "、"Enumeration Comma
The enumeration comma is used to separate list items.      "，" comma is same as English